### PR TITLE
add EFCore.SoftDelete

### PR DESCRIPTION
**Pull request by AshkanAbd**
An implementation for soft deleting entities for EntityFramework Core.

### CODE REVIEW

File: entity-framework/core/extensions/index.md [hunk 1]
```diff
@@ -225,3 +225,9 @@ Alternative translator of LINQ queries to SQL expressions. For EF Core: 3.
 Includes support for advanced SQL features such as CTEs, bulk copy, table hints, windowed functions, temporary tables, and database-side create/update/delete operations.
 
 [GitHub repository](https://github.com/linq2db/linq2db.EntityFrameworkCore)
+
+### EFCore.SoftDelete
+
+An implementation for soft deleting entities for EntityFramework Core.
+
+[GitHub repository](https://github.com/AshkanAbd/efCoreSoftDeletes)
```
Comment (ErikEJ) [inline entity-framework/core/extensions/index.md:233]: Each extension must be available as a NuGet package - where is yours?

File: entity-framework/core/extensions/index.md [hunk 2]
```diff
@@ -225,3 +225,9 @@ Alternative translator of LINQ queries to SQL expressions. For EF Core: 3.
 Includes support for advanced SQL features such as CTEs, bulk copy, table hints, windowed functions, temporary tables, and database-side create/update/delete operations.
 
 [GitHub repository](https://github.com/linq2db/linq2db.EntityFrameworkCore)
+
+### EFCore.SoftDelete
+
+An implementation for soft deleting entities for EntityFramework Core.
```
Comment (ErikEJ) [inline entity-framework/core/extensions/index.md:231]: Please indicated supported EF Core version, like all other entries in the list.

File: entity-framework/core/extensions/index.md [hunk 3]
```diff
@@ -228,6 +228,7 @@ Includes support for advanced SQL features such as CTEs, bulk copy, table hints,
 
 ### EFCore.SoftDelete
 
-An implementation for soft deleting entities for EntityFramework Core.
+An implementation for soft deleting entities for EntityFramework Core. For EF Core 3.1 and later.
```
Comment (ErikEJ) [inline entity-framework/core/extensions/index.md:231]: Just write: 

     For EF Core: 3. 

(Like it is done everywhere else in this file)

File: entity-framework/core/extensions/index.md [hunk 4]
```diff
@@ -228,6 +228,7 @@ Includes support for advanced SQL features such as CTEs, bulk copy, table hints,
...
 
 [GitHub repository](https://github.com/AshkanAbd/efCoreSoftDeletes)
+[Nuget package](https://www.nuget.org/packages/EFCore.SoftDelete/)
```
Comment (ErikEJ) [inline entity-framework/core/extensions/index.md:234]: Remove the GitHub link then (no need for two links)

Comment (ErikEJ) [reply entity-framework/core/extensions/index.md:234]: Ad it is "NuGet"

File: entity-framework/core/extensions/index.md [hunk 5]
```diff
@@ -230,4 +230,4 @@ Includes support for advanced SQL features such as CTEs, bulk copy, table hints,
 
 An implementation for soft deleting entities for EntityFramework Core. For EF Core 3.
 
-[Nuget package](https://www.nuget.org/packages/EFCore.SoftDelete)
+[Nuget](https://www.nuget.org/packages/EFCore.SoftDelete)
```
Comment (ErikEJ) [inline entity-framework/core/extensions/index.md:233]: It is spelled "NuGet" (upper case G)

File: entity-framework/core/extensions/index.md [hunk 6]
```diff
@@ -225,3 +225,9 @@ Alternative translator of LINQ queries to SQL expressions. For EF Core: 3.
 Includes support for advanced SQL features such as CTEs, bulk copy, table hints, windowed functions, temporary tables, and database-side create/update/delete operations.
 
 [GitHub repository](https://github.com/linq2db/linq2db.EntityFrameworkCore)
+
+### EFCore.SoftDelete
+
+An implementation for soft deleting entities for EntityFramework Core. For EF Core 3.
```
Comment (ErikEJ) [inline entity-framework/core/extensions/index.md:231]: Maybe just: "An implementation for soft deleting entities. For EF Core 3."

File: entity-framework/core/extensions/index.md [hunk 7]
```diff
@@ -225,3 +225,9 @@ Alternative translator of LINQ queries to SQL expressions. For EF Core: 3.
 Includes support for advanced SQL features such as CTEs, bulk copy, table hints, windowed functions, temporary tables, and database-side create/update/delete operations.
 
 [GitHub repository](https://github.com/linq2db/linq2db.EntityFrameworkCore)
+
+### EFCore.SoftDelete
+
+An implementation for soft deleting entities. For EF Core 3.
```
Comment (ErikEJ) [inline entity-framework/core/extensions/index.md:231]: Sorry: It should be like everywhere else:

For EF Core: 3.

(My bad)

**Comment by ErikEJ** [comment]
LGTM

**Comment by ajcvickers** [comment]
Thanks @AshkanAbd, and thanks @ErikEJ for the reviews.

**Pull request merged**